src/App/Pages/Accounts/LockPageViewModel.cs

```diff
@@ -149,6 +150,12 @@ public string LockedVerifyText
             set => SetProperty(ref _lockedVerifyText, value);
         }
 
+        public bool CheckPendingAuthRequests
+        {
+            get => _checkPendingAuthRequests;
+            set => SetProperty(ref _checkPendingAuthRequests, value);
+        }
+
```

Comment:
⛏️ There is no need for a property that notify changes. You can use a common property here without explicit backing field:
```suggestion
        public bool CheckPendingAuthRequests { get; set; }
```

Reply:
yup indeed